GameServerLib/Config.cs

```diff
@@ -90,6 +97,30 @@ private void LoadConfig(Game game, string json)
             MapData = ContentManager.GetMapData(GameConfig.Map);
             MapSpawns = ContentManager.GetMapSpawns(GameConfig.Map);
         }
+
+        private string getContentPath()
+        {
+            string result = null;
+            var path = new DirectoryInfo(Directory.GetCurrentDirectory());
```

Comment:
Use `Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);` instead of `Directory.GetCurrentDirectory()` (this may return something different from the exe folder, seems more reliable - see https://stackoverflow.com/questions/15653921/get-current-folder-path)

Reply:
Done. Kept the old one if the new one is null.